access-db.js

```diff
@@ -209,6 +209,60 @@ async function isEmployeeAdmin(userEmail) {
 
 }
 
+async function incrementEquipmentId(equipmentId) {
+    console.log("Querying container: Equipment");
+
+    // query to return the equipmentId of an item
+    const querySpec = {
+        query: "SELECT e.equipmentId FROM Equipment e WHERE e.equipmentId = " + equipmentId
+    };
+
+    // read all items in the Items container
+    const { resources: items } = await equipmentContainer.items
+        .query(querySpec)
+        .fetchAll();
+
+        var latestId = items[0].equipmentId;
+
+        latestId = latestId + 1;
+    
+        items[0].equipmentId = latestId;
+
+    return items[0];
+}
+
+async function giveAdminPriviledge(userEmail) {
+
+    const querySpec = {
+        query: "SELECT e.isAdmin FROM Company c Join e in c.employees Where e.email = '" + userEmail + "'"
+    };
+
+    const { resources: items } = await companyContainer.items
+        .query(querySpec)
+        .fetchAll();
+
+        items[0].isAdmin = true;
```

Comment:
Looks good but we need to push the new isAdmin value to the database, take a look at AddEmployeeToCompany